plugins/scaffolder/api-report.md

```diff
@@ -262,39 +262,6 @@ export const OwnedEntityPickerFieldExtension: FieldExtensionComponent_2<
   }
 >;
 
-// @public (undocumented)
-export const OwnedEntityPickerFieldSchema: FieldSchema<
```

Comment:
@benjdlambert Do these types really need to be exported for any of the extensions? They're not expected to be used are they? As far as I can tell it's all already baked into the `OwnedEntityPickerFieldExtension` type above.

If these are intended for any form of usage we'd need to keep this in place to avoid a breaking change, although we could alias it to the entity picker schema.

Reply:
Correct - they're not meant to be exported I don't think, only the main extension to bake them in the types. I hope that these aren't being used, otherwise we should deprecate them first. Looks like there's some others thats unfortunately been exported too.